pkg/skaffold/deploy/deploy_mux.go

```diff
@@ -103,7 +104,16 @@ func (m DeployerMux) RegisterLocalImages(images []graph.Artifact) {
 	}
 }
 
-func (m DeployerMux) Deploy(ctx context.Context, w io.Writer, as []graph.Artifact, l manifest.ManifestList) error {
+func (m DeployerMux) ConfigName() string {
+	var configNames []string
```

Comment:
I think that `DeployerMux.configName` can just return `""`, since we don't expect it to be invoked, right?

Reply:
Done